src/snapshot.rs

```diff
@@ -153,6 +163,30 @@ impl SnapshotCreator {
     }
   }
 
+  pub unsafe fn get_isolate_ptr(&self) -> *mut Isolate {
+    v8__SnapshotCreator__GetIsolate(self)
+  }
+
+  /// Borrows the Isolate from the SnapshotCreator.
+  pub fn get_isolate(&mut self) -> Locker {
+    unsafe { Locker::new(self.get_isolate_ptr().as_mut().unwrap()) }
+  }
+
+  /// Borrows the Isolate from the SnapshotCreator, also returning a reference
+  /// to the SnapshotCreator.
+  pub fn get_isolate_and_handle(&mut self) -> (Locker, &mut SnapshotCreator) {
```

Comment:
This method is a bit weird, I suppose it is needed to make `SnapshotCreator::set_default_context(context: Local<Context>)` work?
I think it would be better to change the signature to `SnapshotCreator::set_default_context(context: Global<Context>)`, that way it could be called outside a `HandleScope`.

Reply:
I don't see the need for this method. The isolate can be obtained from `get_isolate` directly. And the snapshotcreator itself can be used for `set_default_context` just fine. The only requirement is that the `scope` must go out of scope (dropped) before `create_blob` is called. As far as I can tell, it can be removed.